flink-python/pyflink/table/expression.py

```diff
@@ -1512,6 +1512,19 @@ def array_reverse(self) -> 'Expression':
         """
         return _binary_op("arrayReverse")(self)
 
+    def array_slice(self, start, length) -> 'Expression':
+        """
+        Returns an array that is a subset of the original array, extracted based on the specified
+        `start` index and `length`. Supports both positive and negative indices. Positive indices
+        start from 1 (the first element) while negative indices start from the end of the array
+        (-1 being the last element). If the original array or start or length are null, null is
+        returned. If the `length` exceeds the available elements starting from the `start` index,
+        all elements till the end of the array are returned. If abs(start) is greater than the
+        length of the array or start equal to 0 or length is smaller than 0 or equal to 0, return
+        null.
```

Comment:
Sam comment as above for doc

Reply:
Ok, let me  change behavior of the function.